jobs/integr8ly/poc-testing-executor.yaml

```diff
@@ -195,28 +226,30 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
             }
         }
 
-        def runTests(testPipeline) {
+        def runTests() {
+
             parameters = [
                 string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
                 string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
                 string(name: 'CLUSTER_URL', value: "https://${CLUSTER_HOSTNAME}"),
+                string(name: 'EVALS_USERNAME', value: "${EVALS_USERNAME}"),
+                string(name: 'EVALS_PASSWORD', value: "${EVALS_PASSWORD}"),
                 string(name: 'ADMIN_USERNAME', value: "${OC_USER}"),
                 string(name: 'ADMIN_PASSWORD', value: "${OC_PASSWORD}"),
-                string(name: 'NAMESPACE_PREFIX', value: "${NAMESPACE_PREFIX}")
+                string(name: 'CUSTOMER_ADMIN_USERNAME', value: "${CUSTOMER_ADMIN_USERNAME}"),
+                string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                string(name: 'RELEASE_VERSION', value: "${RELEASE_VERSION}"),
+                string(name: 'NAMESPACE_PREFIX', value: "${NAMESPACE_PREFIX}"),
+                string(name: 'WEBAPP_URL', value: "https://tutorial-web-app-${NAMESPACE_PREFIX}webapp.apps.${YOURCITY}"),
```

Comment:
This doesn't seem to me as a full and correct URL. Is it? I guess something like CLUSTER_HOSTNAME suffix is missing.

Reply:
good catch. There should be CLUSTER_HOSTNAME instead of YOURCITY